# KeyValue.msg

# Key
string key

# Object name
string object_name

# Value
geometry_msgs/Pose value
